svc-bip-api/build.gradle

```diff
@@ -6,8 +6,6 @@ plugins {
 
 ext {
   jacocoExclude = ['**/*model*']
-  // TODO: improve test code coverage so that the following can be increased or removed
-  jacoco_minimum_coverage = 0.4
```

Comment:
can you describe a bit more the motivation for removing this minimum?

Reply:
This minimum overrides the default, lowering it from 0.8 to 0.4. Code coverage is now over 0.8 with this PR, so we want to remove this override.